BraintreeCore/src/test/java/com/braintreepayments/api/AnalyticsClientUnitTest.java

```diff
@@ -319,14 +313,29 @@ public void reportCrash_whenLastKnownAnalyticsUrlExists_sendsCrashAnalyticsEvent
     }
 
     @Test
-    public void reportCrash_whenLastKnownAnalyticsUrlMissing_doesNothing() {
+    public void reportCrash_whenLastKnownAnalyticsUrlMissing_doesNothing() throws JSONException {
+        Configuration configuration = Configuration.fromJson(Fixtures.CONFIGURATION_WITH_ANALYTICS);
+
+        DeviceMetadata metadata = createSampleDeviceMetadata();
+        when(deviceInspector.getDeviceMetadata(context, sessionId, integration)).thenReturn(metadata);
+
+        AnalyticsClient sut = new AnalyticsClient(httpClient, analyticsDatabase, workManager, deviceInspector);
+        sut.sendEvent(configuration, eventName, sessionId, integration, authorization);
```

Comment:
Doesn't `sendEvent` set the `lastKnownAnalyticsUrl`? I'm confused about this test

Reply:
Good call this test is identical to the one below. It's a copy-paste error I'll revert.